bootstrap/README.OpenServer5

```diff
@@ -1,266 +1,28 @@
-	$ PATH=$HOME/tools3/bin:$HOME/tools/bin:$PATH ~/tools/bin/make
-	$ PATH=$HOME/tools3/bin:$HOME/tools/bin:$PATH ~/tools/bin/make install
-
-15. Build GCC 4.2.4 (bootstrap) with patch (*3)
-    Use OpenServer's native LD, GNU binutils (AS), GNU make, and GCC.
-
-	$ mkdir 42build1
-	$ cd 42build1
-	$ PATH=$HOME/tools3/bin:$HOME/tools/bin:$PATH ../gcc-4.2.4/configure --prefix=$HOME/tools4 --with-gnu-as --disable-nls --enable-languages=c
-	$ PATH=$HOME/tools3/bin:$HOME/tools/bin:$PATH $HOME/tools/bin/make bootstrap
-	$ PATH=$HOME/tools3/bin:$HOME/tools/bin:$PATH $HOME/tools/bin/make install
-
-16. Build GNU binutils 2.24 with patch (*4)
-    Use OpenServer's native LD, GNU binutils (AS), GNU make, and GCC.
-
-	$ mkdir 224binutils
-	$ cd 224binutils
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ../binutils-2.24/configure --prefix=/opt/prebuilt --disable-nls --disable-werror
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make
-	# PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make install
-
-17. Build GNU gmp 4.2.4 from gmp-4.2.4.tar.gz
-    Use OpenServer's native LD, GNU binutils (AS), GNU make, and GCC.
+Guidelines
+==========
 
-	$ cd gmp-4.2.4
-	$ PATH=/opt/prebuilt/bin:$PATH ./configure --prefix=/opt/prebuilt --build=i386-pc-sco3.2v5.0.7
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make
-	# PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make install
+First install XDEV as described here:
 
-18. Build mpfr 2.3.2 from mpfr-2.3.2.tar.gz
-    Use OpenServer's native LD, GNU binutils (AS), GNU make, and GCC.
+https://download.xinuos.com/openserver5D/D2M1/packages/5D_Xinuos_Extended_Development_System.pdf
 
-	$ cd mpfr-2.3.2
-	$ PATH=/opt/prebuilt/bin:$PATH ./configure --prefix=/opt/prebuilt
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make
-	# PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make install
+# patchck -o dev
 
-19. Build GNU mpc 0.8.2 from mpc-0.8.2.tar.gz
-    Use OpenServer's native LD, GNU binutils (AS), GNU make, and GCC.
+This gives you the GCC compiler, among other important tools.
 
-	$ cd mpc-0.8.2
-	$ PATH=/opt/prebuilt/bin:$PATH ./configure --prefix=/opt/prebuilt
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make
-	# PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ~/tools/bin/make install
+Ensure XDEV is in the PATH:
 
-20. Build GCC 4.2.4 fullset (bootstrap) with patch (*3)
-    Use OpenServer's native LD, GNU binutils (AS), GNU make, and GCC.
-    After this step, it is ready to bootstrap pkgsrc with GCC 4.2.4.
+# PATH=/usr/xdev/bin:$PATH
+# export PATH
 
-	$ mkdir 42build2
-	$ cd 42build2
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH ../gcc-4.2.4/configure --prefix=/opt/prebuilt --with-gnu-as --disable-nls
-	$ PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH $HOME/tools/bin/make bootstrap
-	# PATH=$HOME/tools4/bin:$HOME/tools/bin:$PATH $HOME/tools/bin/make install
+Remove /usr/lib/libz.la and other libtool archives
```

Comment:
What's the issue here? Having them on Tiger helped expose a buildlink3 bug that I fixed. Would prefer to fix any other issues rather than work around them.

Reply:
"/usr/lib/libz.la is not a valid libtool archive" during bootstrap. I couldn't figure out how it was leaking into the build, there were no references to it in any log file and --prefer-pkgsrc didn't help.